qiskit/visualization/pulse/matplotlib.py

```diff
@@ -676,6 +678,7 @@ def _draw_frequency_changes(self, ax, sf: Dict[int, SetFrequency], y0: float) ->
                 va="center",
                 rotation=90,
             )
+        # TODO: doesn't return anyting?
```

Comment:
Not sure if there is a return missing or wrong type annotation

Reply:
ditto https://github.com/Qiskit/qiskit-terra/pull/8270/files/c6b17c1e774a847fc825dd5956ebfead8adfd1ae#r1111874120